mLRS/CommonTx/setup_tx.h

```diff
@@ -51,7 +51,7 @@
 #define SETUP_OPT_RX_SERIAL_BAUDRATE  "9600,19200,38400,57600,115200,230400"
 #define SETUP_OPT_SERIAL_LINK_MODE    "transp.,mavlink"
 
-#define SETUP_OPT_RF_BAND_LONGSTR     "2.4 GHz,915 MHz FCC,868 MHz" // this is used e.g. in cli
+#define SETUP_OPT_RF_BAND_LONGSTR     "2.4 GHz,915 MHz FCC,868 MHz, 433 MHz, 70 CM HAM" // this is used e.g. in cli
```

Comment:
no blanks before/after ','
70 cm HAM

the problem here is that only 44 char are allowed max ... I guess we need to shorten to 433 and 70 ... in the lua they are faked anyway, and I guess one need to do also in the CLI
so, keep it for teh moment, but you will see me doing something

Reply:
Updated to '2.4 GHz,915 MHz FCC,868 MHz,433,70'